wbooks/src/app/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import logoWolox from './assets/LogoWolox.png';
+import './application.css';
+
+class App extends Component {
+  state = {
+    name: 'Nombre',
+    lastname: 'Apellido',
+    email: 'example@domain.com',
+    password: 'password',
+    password_confirm: 'password'
+  };
+
+  render () {
```

Comment:
The render should be the function in the class

Reply:
done